server/src/models/userModel.ts

```diff
@@ -1,6 +1,6 @@
 import mongoose, { Schema, type Document, type Model } from 'mongoose';
 import myDB from './dbConnection';
-import bycrypt from 'bcrypt';
+import bcrypt from 'bcrypt';
```

Comment:
dang lol, is this all we need to fix the hash issue? can we revert your changes below if this is working ?

Reply:
No, I think the problem is in siginup
Look I just  signed up now and my user data in db looks like 

_id
6584f96987973d18cbe82a52
name
"shezzii"
password"12345678"
...
}
For now I just bypassed this by just commenting preSave hook in model

Reply:
It's recent which is strange. I'm seeing sign ups from a few days ago properly hashed